src/screens/UserPortal/Chat/Chat.tsx

```
@@ -105,8 +153,21 @@
+    },
+  });
+
+  useEffect(() => {
+    markChatMessagesAsRead().then(() => {
+      chatsListRefetch({ id: userId });
+    });
+  }, [selectedContact]);
+
   React.useEffect(() => {
-    if (chatsListData) {
+    if (chatsListData && chatsListData?.chatsByUserId.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)